This addresses a -Werror=missing-braces build failure from GCC bug #53119.

--- a/libdpe/pe_allocspace.c
+++ b/libdpe/pe_allocspace.c
@@ -61,7 +61,7 @@
 	struct pe32plus_opt_hdr *opthdr = pe->state.pe32plus_exe.opthdr;
 
 	Pe_Scn *scn = NULL;
-	struct section_header shdr = { 0, }, tmp_shdr;
+	struct section_header shdr = { { 0, } }, tmp_shdr;
 	if (pehdr->sections < 1)
 		return -1;
 
